instrumentation/opentelemetry-instrumentation-asyncio/src/opentelemetry/instrumentation/asyncio/__init__.py

```diff
@@ -116,17 +116,6 @@ class AsyncioInstrumentor(BaseInstrumentor):
         "run_coroutine_threadsafe",
     ]
 
-    def __init__(self):
-        super().__init__()
-        self.process_duration_histogram = None
-        self.process_created_counter = None
-
-        self._tracer = None
-        self._meter = None
-        self._coros_name_to_trace: set = set()
-        self._to_thread_name_to_trace: set = set()
-        self._future_active_enabled: bool = False
```

Comment:
Is the related to the issue or just some old dead code?

Reply:
I removed it because it's not related to the issue and I don't think it's needed, and I thought it was right to remove it because sometimes when I run the test code, the process_duration_histogram and process_created_counter are None.